content/docs/self-hosted-runners.md

```diff
@@ -104,17 +101,40 @@ train-model:
   script:
     - pip install -r requirements.txt
     - python train.py
-  image: iterativeai/cml:0-dvc2-base1
-  script:
-    - pip install -r requirements.txt
-    - python train.py
-
     # Create CML report
     - cat metrics.txt >> report.md
     - cml publish plot.png --md --title="Confusion Matrix" >> report.md
     - cml send-comment report.md
 ```
 
+</tab>
+<tab title="Bitbucket">
+
+```yaml
+pipelines:
+  default:
+    - step:
+        image: iterativeai/cml:0-dvc2-base1
+        script:
+          - |
+            cml runner \
+                --cloud=aws \
+                --cloud-region=us-west \
+                --cloud-type=p2.xlarge \
+                --cloud-spot \
+                --labels=cml-gpu
+    - step:
+        runs-on: [self.hosted, cml-gpu]
+        image: iterativeai/cml:0-dvc2-base1-gpu
```

Comment:
do GPUs actually work on BB?

Reply:
No, please see the callout on https://github.com/iterative/cml.dev/pull/243#issue-1246387011

Reply:
Should we add a warning and/or remove mentions to GPU until https://github.com/iterative/cml/issues/1015 is resolved?

Reply:
I'd say:

```diff
- --labels=cml.gpu
- --cloud-type=p2.xlarge
- image: iterativeai/cml:0-dvc2-base1-gpu
+ # gpu not yet supported, see https://github.com/iterative/cml/issues/1015
```

Reply:
https://github.com/iterative/cml.dev/pull/243#discussion_r882186255